File: call-in-script/0.simple-script.pl
#!/usr/bin/env perl
use strict;
use warnings;
$ENV{SAC_DISPLAY_COPYRIGHT}=0;

open(SAC, "| sac ") or die "Error opening sac\n";
print SAC "fg seismo \n";
print SAC "lh evla kstnm \n";
print SAC "q \n";
close(SAC);

File: call-in-script/0.simple-script.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

import os
import subprocess

os.putenv("SAC_DISPLAY_COPYRIGHT", '0')

p = subprocess.Popen(['sac'], stdin=subprocess.PIPE)
s = "fg seismo \n"
s += "lh evla kstnm \n"
s += "q \n"
p.communicate(s.encode())
